pkg/v1/layout/testdata/test_image/555b1001d54ed229f56990845856f080b0707348b26b3aa85aaf58d5570cdee0/VERSION

```diff
@@ -0,0 +1 @@
+1.0
```

Comment:
Can you document in `testdata/test_image/README` how to regenerate these in the future?

Reply:
If only I knew.